.infrastructure/55_cognito.tf

```
@@ -271,6 +271,18 @@ resource "aws_cognito_user_pool" "devportal" {
     }
   }
 
+  schema {
+    name                     = "user_preferences"
+    attribute_data_type      = "String"
+    developer_only_attribute = false
+    mutable                  = true
+    required                 = false
+    string_attribute_constraints {
+      min_length = 1
+      max_length = 131072
```

```suggestion
      max_length = 2048
```
Max length must be 2048. [Check this error out](https://github.com/pagopa/developer-portal/actions/runs/6929410766/job/18847057174#step:9:293)